cmake/QskFindMacros.cmake

```diff
@@ -24,6 +24,8 @@ macro(qsk_setup_Qt)
     else()
         message(FATAL_ERROR "Couldn't find any Qt package !")
     endif()
+    
+    find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Quick)
```

Comment:
There is already a find_package( ... Quick) at the beginning of qsk_setup_Qt.
Could you please find out why it is not working in your case ?

Reply:
Yes it is strange, and the patch effectively fixes the issue in my case. I can reproduce both on windows / vcpkg / qt 6 and with Arch Linux / Qt 6.7.1.

I've found this:
[https://doc.qt.io/qt-6/cmake-qt5-and-qt6-compatibility.html#supporting-older-qt-5-versions](https://doc.qt.io/qt-6/cmake-qt5-and-qt6-compatibility.html#supporting-older-qt-5-versions)

They suggest that for older qt 5 versions should call find_package twice, one with the QT + NAMES and the second with the effective version. In my case it is needed even with Qt6